src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -38,6 +38,7 @@ def serialize(self, obj, attrs, user):
             "aggregation": obj.aggregation,
             "aggregations": [obj.aggregation],
             "timeWindow": obj.time_window,
+            "environment": obj.environment,
```

Comment:
There are multiple environments here, so you'll need to fetch them for all alert rules in one query in `get_attrs` and have them as a list here. Are you just going to return the env names here?

Reply:
It sends back the env id's now, just as they are sent in.

Reply:
Damn, I missed this earlier. We usually try to pass things like environments/projects via slug rather than ID. We haven't always been consistent on this, but it'd be better to use the environment slug/name here.